src/ui/src/pages/credits/credits.tsx

```diff
@@ -21,11 +21,14 @@ import * as React from 'react';
 import { Button } from '@mui/material';
 import { Theme } from '@mui/material/styles';
 import { createStyles, makeStyles } from '@mui/styles';
+import Axios from 'axios';
 
 import { buildClass, scrollbarStyles, Footer } from 'app/components';
+import { PIXIE_CLOUD_VERSION } from 'app/utils/env';
 import { WithChildren } from 'app/utils/react-boilerplate';
 import { Copyright } from 'configurable/copyright';
-import licenseJson from 'configurable/licenses.json';
+
+const licenseURL = `https://storage.googleapis.com/pixie-dev-public/oss-licenses/${PIXIE_CLOUD_VERSION.tag}.json`;
```

Comment:
This will fail for airgapped users, leaving them without license information. Is that acceptable?

Reply:
Yeah I thought about this issue already ... I have a different approach I'm going to try, hold on.